docs/dynamic-plugins.md

```diff
@@ -988,3 +992,63 @@ The required options mirror the [AppTheme](https://backstage.io/docs/reference/c
 - `variant` Whether the theme is `light` or `dark`, can only be one of these values.
 - `icon` a string reference to a system or [app icon](#extend-internal-library-of-available-icons)
 - `importName` name of the exported theme provider function, the function signature should match `({ children }: { children: ReactNode }): React.JSX.Element`
+  
+
+## Debugging Dynamic Plugins
+
+### Backend Dynamic Plugins Local Debug
+
+For local debugging of Dynamic Plugins you need to clone `backstage-showcase` and run it ready for debugging and attach your IDE debugger to the backend process. First it is required to build and copy the dynamic plugin :
+
+* Build your plugin and export the dynamic package
+```
+cd ${pluginRootDir}
+yarn build && yarn run export-dynamic
+```
+* Copy the resulting `dist-dynamic` directory to dynamic-plugins-root/${plugin-id}
+
+Once the plugin is built and deployed, it is time to prepare the showcase to run in debug mode:
+
+* Start backstage-showcase backend on a terminal:
+   * Go to packages/backend
+   * Run yarn run start –inspect
```

Comment:
need another `-` for `--inspect`.  Also an alternative to cd'ing into the backend package is to run `yarn workspace backend start --inspect` which I think should work the same way.